recipes/sos-sas/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "sos-sas" %}
+{% set version = "0.18.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8611cb450f86f2d07b74ea1043ec0bc40fb1b19a98c8c7aa52db889b12211891
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . -vv
+  noarch: python
```

Comment:
you don't need noarch python twice :wink:

Reply:
Thanks. Not sure how it got there.